venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -35,51 +35,223 @@ def process(client, note, invitation):
 Thanks!
 
 OpenReview Team
-            '''.format(noteId = forum.id, conference_id = conference.get_id(), program_chairs_id = conference.get_program_chairs_id())
+            '''.format(baseurl = client.baseurl, noteId = forum.id, conference_id = conference.get_id(), program_chairs_id = conference.get_program_chairs_id())
         }
     )
     client.post_note(comment_note)
 
     readers = [conference.get_program_chairs_id(), 'OpenReview.net/Support']
-    revision_invitation = client.get_invitation(id= 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Revision')
-    revision_invitation.reply['readers'] = {
-        'values':  readers
-    }
-    revision_invitation.invitees = readers
-    client.post_invitation(revision_invitation)
+    readers.extend(forum.signatures[:])
 
     forum.writers = ['OpenReview.net']
+    forum.readers = readers
     client.post_note(forum)
 
-    reviewer_recruitment_invitation = client.post_invitation(openreview.Invitation(
-        id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Reviewer_Recruitment',
-        super = 'OpenReview.net/Support/-/Reviewer_Recruitment',
+    revision_invitation = client.post_invitation(openreview.Invitation(
+        id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Revision',
+        super = 'OpenReview.net/Support/-/Revision',
         invitees = readers,
         reply = {
             'forum': forum.id,
-            'replyto': forum.id,
+            'referent': forum.id,
             'readers' : {
                 'description': 'The users who will be allowed to read the above content.',
                 'values' : readers
             }
         },
-        signatures = ['OpenReview.net/Support']
+        signatures = [conference.get_program_chairs_id()]
```

Comment:
The signatures here dictate the permissions that the process function of revision will run with and we want all process functions to run with PC permissions so that they get access only within their conference space. OpenReview.net/Support can still these processes.